test/test-caching.js

```diff
@@ -1,5 +1,5 @@
-// Copyright IBM Corp. 2016,2017. All Rights Reserved.
-// Node module: loopback
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
```

Comment:
2016,2018

Reply:
Actually i'm not sure why the tool changed it to `2017`. but can change it back to 2016.